icu_benchmarks/models/custom_metrics.py

```
@@ -143,3 +144,68 @@
+
+def sensitivity(y_preds: torch.Tensor | ndarray, y_targets: torch.Tensor | ndarray) -> float:
+    if isinstance(y_preds, torch.Tensor):
+        y_preds = y_preds.numpy()
+    if isinstance(y_targets, torch.Tensor):
+        y_targets = y_targets.numpy()
+    y_true = np.rint(y_targets).astype(int)
+    y_pred = np.rint(y_preds).astype(int)
+    tn, fp, fn, tp = sk_confusion_matrix(y_true, y_pred).ravel()
+    return tp / (tp + fn)
+
+class Specificity(EpochMetric):
```

Ruff (0.8.2)

Redefinition of unused `Specificity` from line 10

(F811)